packages/store/tests/utils/angular.spec.ts

```diff
@@ -1,82 +1,73 @@
-import { platformBrowserDynamic } from '@angular/platform-browser-dynamic';
-import { getPlatform, platformCore } from '@angular/core';
-import { platformBrowser } from '@angular/platform-browser';
-import {
-  platformBrowserDynamicTesting,
-  BrowserDynamicTestingModule
-} from '@angular/platform-browser-dynamic/testing';
-import { getTestBed } from '@angular/core/testing';
+import { ɵglobal } from '@angular/core';
+
 import { isAngularInTestMode } from '@ngxs/store/internals';
 
 describe('[utils/angular]', () => {
   describe('isAngularInTestMode', () => {
-    function resetEnv() {
+    // Just an empty object so `typeof !== undefined` will be truthy.
+    const nonUndefinedValue = {};
+
+    function resetMemoization() {
       const fn = <any>isAngularInTestMode;
       // Reset the memoization
       fn && fn.reset && fn.reset();
```

Comment:
I don't think it is necessary to memoise this anymore because the checks are so simple!
We could remove the memoisation from the lib export and remove this testing code.

Reply:
Removed memoization.